bCNC/controllers/GRBL1.py

```diff
@@ -8,6 +8,8 @@
 import time
 import Utils
 
+OV_JOG_CANCEL   = chr(0x85)
```

Comment:
Could you remove the extra spaces here. To silence codefactor.

Reply:
Already fixed in my local version, need to push the change